CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 86 ending 2018-05-21
+
+### Added
+- Delete API for transformation_mappings [(#383)](https://github.com/ManageIQ/manageiq-api/pull/383)
+- Adds request_retirement start [(#380)](https://github.com/ManageIQ/manageiq-api/pull/380)
```

Comment:
can you update to just Adds request_retire action for Vms and Services ?
Thanks,